eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -268,13 +268,13 @@ void stopsOnCritical(@TempDir final Path temp) throws Exception {
                     .result()
                     .get(
                         String.format(
-                            "target/%s/foo/x/main/02-duplicate-names.xml",
+                            "target/%s/foo/x/main/26-duplicate-names.xml",
```

Comment:
@Graur To be honest, I don't clearly understand why `26`. Does it some sort of the order?

Reply:
@volodya-lombrozo Yes, it's something like `xsl` file execution order